dom/package.json

```diff
@@ -33,6 +33,17 @@
   "module": "lib/es6/index.js",
   "typings": "lib/cjs/index.d.ts",
   "types": "lib/cjs/index.d.ts",
+  "scripts": {
+    "build": "pnpm run build-cjs && pnpm run build-es6",
```

Comment:
TBH, there is currently no difference between `npm run` and `pnpm run`. I guess for consistency it is better to use pnpm but no other reason at the moment